app/views/layouts/govuk_publishing_components/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <%= csrf_meta_tags %>
     <%= favicon_link_tag "govuk_publishing_components/favicon-production.png" %>
 
-    <%= stylesheet_link_tag "component_guide/application", media: "screen" %>
+    <%= stylesheet_link_tag "component_guide/application", media: "all" %>
     <%= stylesheet_link_tag "component_guide/print", media: "print" %>
```

Comment:
Should we get rid of this too? I think it doesn't do anything.

Reply:
Good shout - I've just pushed another commit to remove this and a range of other references to print stuff. Not quite all of it yet though - still pondering whether to remove all the print style files as you suggest and just have the breaking change. Thanks for having a look 👍

Reply:
Think we're going to go with the breaking change as suggested @kevindew. We'll hold off merging this for today so we can get a good run at it on Monday, when we can update individual apps as dependabot warrants. Fairly sure tests will fail without the print styles so it shouldn't be deployable.